standard/expressions.md

```diff
@@ -3213,96 +3213,15 @@ The predefined multiplication operators are listed below. The operators all comp
-    <tr>
-      <td></td>
-     <td><b><code>+y</code></b></td>
-      <td><b><code>-y</code></b></td>
-      <td><b><code>+0</code></b></td>
-      <td><b><code>-0</code></b></td>
-      <td><b><code>+∞</code></b></td>
-      <td><b><code>-∞</code></b></td>
-      <td><b><code>NaN</code></b></td>
-    </tr>
-  <!-- md equivalent: `+x`  | `+z`  | `-z`  | `+0`  | `-0`  | `+∞`  | `-∞`  | `NaN` -->
-    <tr>
-      <td><code>+x</code></td>
-      <td><code>+z</code></td>
-      <td><code>-z</code></td>
-      <td><code>+0</code></td>
-      <td><code>-0</code></td>
-      <td><code>+∞</code></td>
-      <td><code>-∞</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  <!-- md equivalent: `-x`  | `-z`  | `+z`  | `-0`  | `+0`  | `-∞`  | `+∞`  | `NaN` -->
-    <tr>
-      <td><code>-x</code></td>
-      <td><code>-z</code></td>
-      <td><code>+z</code></td>
-      <td><code>-0</code></td>
-      <td><code>+0</code></td>
-      <td><code>-∞</code></td>
-      <td><code>+∞</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  <!-- md equivalent:   `+0`  | `+0`  | `-0`  | `+0`  | `-0`  | `NaN` | `NaN` | `NaN` -->
-    <tr>
-      <td><code>+0</code></td>
-      <td><code>+0</code></td>
-      <td><code>-0</code></td>
-      <td><code>+0</code></td>
-      <td><code>-0</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  <!-- md equivalent: `-0`  | `-0`  | `+0`  | `-0`  | `+0`  | `NaN` | `NaN` | `NaN` -->
-    <tr>
-      <td><code>-0</code></td>
-      <td><code>-0</code></td>
-      <td><code>+0</code></td>
-      <td><code>-0</code></td>
-      <td><code>+0</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  <!-- md equivalent: `+∞`  | `+∞`  | `-∞`  | `NaN` | `NaN` | `+∞`  | `-∞`  | `NaN` -->
-    <tr>
-      <td><code>+∞</code></td>
-      <td><code>+∞</code></td>
-      <td><code>-∞</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>+∞</code></td>
-      <td><code>-∞</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  <!-- md equivalent: `-∞`  | `-∞`  | `+∞`  | `NaN` | `NaN` | `-∞`  | `+∞`  | `NaN` -->
-    <tr>
-      <td><code>-∞</code></td>
-      <td><code>-∞</code></td>
-      <td><code>+∞</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>-∞</code></td>
-      <td><code>+∞</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  <!-- md equivalent: `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` -->
-    <tr>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  </table>
+  |       | `+y`  | `-y`  | `+0`  | `-0`  | `+∞`  | `-∞`  | `NaN` |
```

Comment:
Please could you remove the Custom Word conversion comment? I *suspect* it's now not being used, but it's slightly confusing to have it there :)

Reply:
I did do that, and the Word output was fine. That's now pushed.